admin/v1/admin.proto

```diff
@@ -33,6 +34,16 @@ service Admin {
       get: "/admin/v1/acctgroups/{id}"
     };
   }
+
+  // Update the features available to a user on an Alphaus product. Currently,
+  // only values of "wave" and "aqua" are supported for {product}. For a list of
+  // valid feature flags, see our documentation at https://alphauslabs.github.io/blueapi/apis/admin.html.
+  rpc UpdateFeatureFlags(UpdateFeatureFlagsRequest) returns (google.protobuf.Empty) {
+    option (google.api.http) = {
+      put: "/iam/v1/users/{id}/features/{product}"
```

Comment:
@Woody1193 Admin service is `/admin/v1`. Iam is a separate service.

Reply:
Ah, good catch. I originally had this function in IAM and moved it because it fit Admin better. I'll fix it.